website/api/helpers/strings/to-html.js

```diff
@@ -62,6 +62,7 @@ module.exports = {
 
   fn: function(inputs, exits) {
     var marked = require('marked');
+    var _ = require('lodash');
```

Comment:
Please remove this `require`.  Why?  Twofold:

- Sails exposes Lodash globally by convention in Sails apps unless it is disabled, so better to be consistent and use it in userland backend code  (never in Sails core libs though)
- The version of Lodash exposed by Sails for userland is actually 3.10.x, and it is technically [`@sailshq/lodash`](https://www.npmjs.com/package/@sailshq/lodash), a different package (due to security patches we've done over the years when I took 3.10.x over from @jdalton for security maintenance).  So if you were to `require('lodash')` here, then `lodash` would need to be a dependency in the package.json file of the Fleet website.  Technically this works because there are other copies of `lodash` floating around in node_modules, and (lamentably) Node's require actually can require deeply nested dependencies, rather than only letting you require the direct dependencies declared in your package.json file.  So anyway, in this case, the `lodash` being referenced is actually a different version, likely 4.x.  This happens to work out since `_.kebabCase()` exists in Lodash 4, but it wouldn't always work out.